functions/src/wort/core.clj

```diff
@@ -2,6 +2,16 @@
   (:gen-class
    :methods [^:static [handler [String] String]]))
 
+(require '[clojure.string :as str])
+
 (defn -handler
   [s]
   (str "Wort " s " ツ"))
+
+(defn get_words_from_key
+  [payload key]
+  (let [value (filter
+                #(>= (count %) 1) (str/split (get payload key) #"\s+|[.,!?]+" ))]
```

Comment:
Huh. Yeah that's really strange. No worries then. You could simplify the regex by removing the greedy quantifiers (makes it slightly simpler to read) and use #(not (empty %)) as your filter function. But those are optional.